Script/editUser.js

```diff
@@ -54,4 +54,27 @@ function checkUserDup()
         })
         .catch((response) => {})
     }
-}
\ No newline at end of file
+}
+
+$(".custom-file-input").on("change", function() {
+var fileName = $(this).val().split("\\").pop();
+$(this).siblings(".custom-file-label").addClass("selected").html(fileName);
+});
+
+function validateImage() {
+    var formData = new FormData();
+ 
+    let Imagefile = document.getElementById("customFile").files[0];
+ 
+    formData.append("Filedata", Imagefile);
+    let t = Imagefile.type.split('/').pop().toLowerCase();
+    if (t != "jpeg" && t != "jpg" && t != "png" && t != "bmp" && t != "gif") {
+        alert('Please select a valid image file'); // will add notify.js instead, not sure how to do so leaving this for the moment.
```

Comment:
Look into head.php. Added a function (No need to copy it, its available globally) which allows you to make notification 
Should be Overhang("type", "message")

Reply:
Or perhaps you can add a <small> like I did for username validation? change your username to 'od' and view the validation I used